jpos/src/main/java/org/jpos/space/JDBMSpace.java

```diff
@@ -59,11 +60,23 @@
      * @param filename underlying JDBM filename
      */
     protected JDBMSpace (String name, String filename) {
+        this(name, filename, true);
+    }
+
+    /**
+     * protected constructor.
+     * @param name Space Name
+     * @param filename underlying JDBM filename
+     * @param sync sync transactions to disk
+     */
+    protected JDBMSpace (String name, String filename, boolean sync) {
         super();
         this.name = name;
         try {
             Properties props = new Properties();
             props.put (RecordManagerOptions.CACHE_SIZE, "512");
+            if (!sync)
+                props.put (RecordManagerOptions.DISABLE_TRANSACTIONS, "true");
```

Comment:
The JDBMSpace rely on having transactions. We don't want users to disable that feature, for their protection. If what we want is just letting the tests pass, we can just go ahead and disable the tests altogether.